korgan/claim_exemplar_architecture.py

```
@@ -0,0 +1,176 @@
+        current = {
+            "title": draft.title, "court": draft.court, "claimant": draft.claimant, "defendant": draft.defendant,
+            "price_of_claim": draft.price_of_claim, "facts": draft.facts, "legal_basis": draft.legal_basis,
+            "requests": draft.requests, "attachments": draft.attachments, "verification_notes": draft.verification_notes,
+        }
+        payload = await self._quality_repair(
+            schema_name="korgan_exemplar_architecture_repair", schema=_CLAIM_SCHEMA, case_context=enriched,
+            research=research, current_payload=current, issues=issues, language=language,
+            document_label="исковое заявление по эталонной архитектуре 12 реальных исков",
+            extra_rules=(
+                "13. Исправь только указанные разрывы архитектуры, не создавая новых фактов или способов защиты.\n"
+                "14. Если мотивировка не поддерживает самостоятельное требование — удали нерелевантный блок; если требование явно запрошено и подтверждено — восстанови его в ПРОШУ СУД.\n"
```

Ruff (0.16.2)

[warning] String contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF001)